### DISCUSSION TITLE
Enable Quick App Pairing with GoPro labs

### DISCUSSION THREAD

**Discussion by leSullivan** [gopro/labs#713] [Q&A] [answered]
Cheers. 

i have a GoPro Hero 9 with a broken screen. 
Today  i installed GoPro Labs on my Device to pair it with my phone via QR Code. 
My GoPro could successfully read the QR Code where i selected the Quick App Pairing mode but still my app couldnt recognize the cameral. Any tips on how to solve this ?

**Comment by dnewman-gpsw** [comment, marked as answer]
As you can't see a rear screen, you are likely not seeing the request for button presses. When scanning !PA you must immediately press shutter three times quickly, this is to improve security.

**Reply by leSullivan** [reply]
Omg. Thank you so much for your help!
